components/pickers/image.tsx

```diff
@@ -122,9 +149,10 @@ export default function ImagePicker({
             )}
           </div>
           <Toggle
+            disabled={uploading}
             name="Stretch to Fit"
             description="Stretch your image to fit the width of banner box. The way an image stretchs will vary based on device screen size. "
-            setEnabled={setFill}
+            setEnabled={setStretch as StateUpdater<boolean>}
```

Comment:
if this is now called stretch why is it called filled in the db?

Reply:
need multiple names